src/Integration/IProjectPropertyManager.cs

```diff
@@ -0,0 +1,25 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="IProjectPropertyManager.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using EnvDTE;
+using System.Collections.Generic;
+
+namespace SonarLint.VisualStudio.Integration
+{
+    public interface IProjectPropertyManager
```

Comment:
Public interface - documentation is missing (see my comments around ProjectPropertyManager first since I think that the API needed to be tweaked)